cmd/helm-operator/main.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/operator-framework/operator-sdk/pkg/helm"
+	hoflags "github.com/operator-framework/operator-sdk/pkg/helm/flags"
+	"github.com/operator-framework/operator-sdk/pkg/log/zap"
+)
+
+func main() {
+	logf.SetLogger(zap.Logger())
+
+	flags := setDefaultFlags()
+
+	err := helm.Run(flags)
+	if err != nil {
+		log.Error(err, "Error running helm operator binary")
+	}
+}
```

Comment:
Same scenario of https://github.com/operator-framework/operator-sdk/pull/3331/files#r450745444